lib/swig-i18n.js

```diff
@@ -2,6 +2,8 @@ var _       = require('underscore');
 var console = require('console');
 var swig    = require('swig');
 
+var translations = {};
```

Comment:
moving the object to the module scope, instead of the initialization function scope